packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -910,6 +914,63 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * POST /internal/patient/settings
+ *
+ * Creates or updates patient settings for a select list of patient IDs.
+ *
+ * @param req.query.cxId The customer ID.
+ * @param req.query.patientIds List of patient IDs to update.
+ * @param req.query.adtSubscription Whether to enable or disable ADT subscription. Optional, defaults to false.
+ * @returns 200 with the results of the operation.
+ */
+router.post(
+  "/settings",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const patientIds = getFromQueryAsArrayOrFail("patientIds", req);
+    const adtSubscription = getFromQueryAsBoolean("adtSubscription", req) ?? false;
```

Comment:
Another instance of us defining the default value in multiple places. Let's prefer to do that on the command, please.